server/lib/notifications/agents/discord.ts

```diff
@@ -143,104 +181,57 @@ class DiscordAgent
         }
       );
 
-      if (payload.issue.media.mediaType === MediaType.TV) {
-        fields.push({
-          name: 'Affected Season',
-          value:
-            payload.issue.problemSeason > 0
-              ? `Season ${payload.issue.problemSeason}`
-              : 'All Seasons',
-        });
-
-        if (payload.issue.problemSeason > 0) {
-          fields.push({
-            name: 'Affected Episode',
-            value:
-              payload.issue.problemEpisode > 0
-                ? `Episode ${payload.issue.problemEpisode}`
-                : 'All Episodes',
-          });
-        }
+      switch (type) {
+        case Notification.ISSUE_CREATED:
+        case Notification.ISSUE_REOPENED:
+          color = EmbedColors.RED;
+          break;
+        case Notification.ISSUE_COMMENT:
+          color = EmbedColors.ORANGE;
+          break;
+        case Notification.ISSUE_RESOLVED:
+          color = EmbedColors.GREEN;
+          break;
       }
```

Comment:
Why two different switch cases for this? You can combine this with the other switch case that sets colors?

Reply:
I am handling request notifications and issue notifications separately.